frontend/css/wi-volunteer-management-public.css

```diff
@@ -80,4 +80,33 @@
 
 #wivm-sign-up-form .field-error {
 	border: 2px solid #990000;
+}
+
+/* Widget Styles */
+.widget-volunteer-opp-info {
```

Comment:
I'd say yes. Let's show all of the content on the initial page load. If it's too tall, the admin can always show fewer opportunities.